modules/nw-dns-cache-tuning.adoc

```diff
@@ -33,8 +38,26 @@ spec:
 +
 <1> The string value `1h` is converted to its respective number of seconds by CoreDNS. If this field is omitted, the value is assumed to be `0s` and the cluster uses the internal default value of `900s` as a fallback.
 <2> The string value can be a combination of units such as `0.5h10m` and is converted to its respective number of seconds by CoreDNS. If this field is omitted, the value is assumed to be `0s` and the cluster uses the internal default value of `30s` as a fallback.
+
+.Verification
+
+. To review the change, look at the config map again by running the following command:
 +
-[WARNING]
-====
-Setting TTL fields to low values could lead to an increased load on the cluster, any upstream resolvers, or both.
-====
+[source,terminal]
+----
+oc get configmap/dns-default -n openshift-dns -o yaml
+----
++
+You should see entries that look like this for the changes made above:
```

Comment:
Since you used `.` for the first step, this should also be a numbered step (for example, "Verify that you see entries that look like the following example:"). Otherwise, use an asterisk for the first step. 

IBM Style recommends not using "above, below, etc." to refer to locations in the doc.